app/perftools/management/commands/create_page_cache.py

```diff
@@ -70,6 +70,7 @@ def create_grant_clr_cache():
     pks = Grant.objects.filter(active=True, hidden=False).values_list('pk', flat=True)
     for pk in pks:
         grant = Grant.objects.get(pk=pk)
+        grant.in_active_clrs.clear()
```

Comment:
i think this should be a part of the `calc_clr_round` function so it can be called from anywhere with same results